frontend/components/Dashboard/Dashboard.tsx

```diff
@@ -3,12 +3,14 @@ import { useRouter } from 'next/router'
 import Nav, { navConstants } from './Nav'
 import Header from './Header'
 import useWindowSize from '../../hooks/useWindowSize'
+import { User } from '../../generated/graphql'
```

Comment:
```suggestion
import { User as UserType } from '../../generated/graphql'
```

Reply:
^ just in accordance with the convention we've been following everywhere else

Reply:
I actually was under the impression that our convention was using the name as it is generated from the graphql file and only adding the `as XType` alias when it conflicted with another variable name. I can change it back